test/Turn-test.js

```diff
@@ -9,23 +9,82 @@ describe('Turn', function() {
     it('should be a function', function() {
         const turn = new Turn();
         expect(Turn).to.be.a('function');
-    })
+    });
 
     it('should be an instance of Turn', function() {
         const turn = new Turn();
         expect(turn).to.be.an.instanceOf(Turn);
-    })
+    });
+
+    it('user guess should be a string', function() {
```

Comment:
Nice job updating the file so that it checks that the input value is the appropriate data type.